release/templates/featuresenabled-azuredeploy.json

```diff
@@ -62,7 +55,7 @@
     "variables": {
         "loweredServiceName": "[toLower(parameters('serviceName'))]",
         "featuresEnabledServiceName": "[concat(variables('loweredServiceName'),'-featuresenabled')]",
-        "appServicePlanResourceId": "[resourceId(parameters('appServicePlanResourceGroup'), 'Microsoft.Web/serverfarms/', parameters('appServicePlanName'))]",
```

Comment:
I suppose this means we're always targeting an app service plan that's  in the same resource group as everything else now?